exporter/opentelemetry-exporter-opencensus/setup.cfg

```diff
@@ -46,6 +46,7 @@ install_requires =
     opentelemetry-api ~= 1.3
     opentelemetry-sdk ~= 1.3
     protobuf >= 3.13.0
+    setuptools >= 16.0
```

Comment:
does opencensus exporter use setuptools directly?

Reply:
`pkg_resources` is imported here for opencensus:
https://github.com/open-telemetry/opentelemetry-python/blob/main/exporter/opentelemetry-exporter-opencensus/src/opentelemetry/exporter/opencensus/util.py#L19

Reply:
FWIW, this should list every `setup.cfg` that is missing an explicit `setuptools` requirement:

```sh
grep --include=*.py -rlE 'pkg_resources|setuptools' . \
| sed -nE 's|(.*/)src/.*|\1setup.cfg|p' \
| uniq \
| xargs -n1 bash -c 'grep -q setuptools "$0" || echo "$0"'
```

Reply:
Interesting. I think it should just import opentelemetry-api to get the version but probably that can be fixed independently.